lib/iris/util.py

```diff
@@ -1222,6 +1223,8 @@ def as_compatible_shape(src_cube, target_cube):
 
     .. note:: This function will load and copy the data payload of `src_cube`.
 
+    .. deprecated:: 3.0.0
+
```

Comment:
According to [contributing_deprecations](https://github.com/SciTools/iris/blob/80a887877fed618b20755ec420d8fc46390f1a59/docs/iris/src/developers_guide/contributing_deprecations.rst), you should also include some prose in the docstring:

>As with the deprecation warning, you should include advice on how to
   avoid using the deprecated API within the content of this directive.
   Feel free to include more detail in the updated docstring than in the
   deprecation warning.

Reply:
Good call... I'm just being lazy :sleepy: